src/requirements.ts

```diff
@@ -74,8 +74,12 @@ function checkJavaVersion(javaHome: string): Promise<number> {
       const javaExecutable = path.resolve(javaHome, 'bin', JAVA_FILENAME);
       cp.execFile(javaExecutable, ['-version'], {}, (error, stdout, stderr) => {
             const javaVersion = parseMajorVersion(stderr);
-            if (javaVersion < 8) {
-                rejectWithDownloadUrl(reject, `Java 8 or newer is required to run this extension. Java ${javaVersion} was found at ${javaHome}. Please download and install a recent JDK`);
+            if (!javaVersion) {
+                rejectWithDownloadUrl(reject, `Java 8 or newer is required. No Java was found on your system..
+                Please get a recent JDK or configure if for "Servers View" if it already exists`);
```

Comment:
Is this a typo? "Please get a recent JDK or configure if for..."   ?   "if for" ? 

Same for next else-if as well.

Reply:
@robstryker fixed.